tasks/main.yml

```diff
@@ -50,3 +50,19 @@
   with_items: "{{ sonar_plugins }}"
   loop_control:
     loop_var: "sonar_cplugin"
+
+- meta: flush_handlers
+
+- name: "Ensure application started"
```

Comment:
please rename that step, as it does not wait for server start but insteat directly check the version of sonar via rest API.

Reply:
Ansible check the page every 5 seconds until API reachable.

Would it be ok if I'll propose the name "Wait for API and verify version of started server"?

I'll be appreciate if you confirm the new name or propose the more suitable one - I will push new change after this confirmation.

Thanks!